README.md

```diff
@@ -75,5 +75,7 @@ Tokite support only below events now.
 <tr><td>user:</td><td>Match user name.</td><td>user:hogelog</td></tr>
 <tr><td>label:</td><td>Match pull_request or issue label.</td><td>label:Feature</td></tr>
 <tr><td>review_state:</td><td>Match pull_request_review state.</td><td>review_state:/commented|approved|changes_requested/</td></tr>
+<tr><td>requested_reviewer:</td><td>Match user name of review requested reviewer</td><td>requested_reviewer:hogelog</td></tr>
+<tr><td>requested_team:</td><td>Match team slug of review requested team</td><td>requested_team:chef</td></tr>
```

Comment:
+1 for documentation.

It may be confusing that GitHub Team have two names, `name` and `slug`. `name` is shown at a reviewers list in a pull request page, and `slug` is used by mentions. Also, according to docs <https://docs.github.com/developers/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=review_requested#pull_request>, `slug` can be null in a response field.

I also prefer using `slug` here, because matching `slug` is a bit easier than matching `name` since `name` can contain some special characters such as spaces.

So it is very nice to write "slug" here.

Reply:
Replace `slug` to `name` in doc. https://github.com/cookpad/tokite/pull/53/commits/f3ced22b589bd3704088ce5ecef6e7070258f98e